### PR TITLE
Added doc page how to setup yast on firstboot

### PR DESCRIPTION
**Pull request by schaefi**
Document how to setup yast in the KIWI image description so that it is called on first boot of the image. This is the result of a thread on the mailing list

### CODE REVIEW

**Comment by schaefi** [comment]
@tomschr if you have a second would be great to get your feedback

File: doc/source/building/working_with_images/setup_yast_on_first_boot.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.. _yast-on-boot:
+
+Setting Up YAST at First Boot
+=============================
+
+.. sidebar:: Abstract
+
+    This page provides information how to setup the KIWI
+    XML description to start the SUSE YAST system setup
```
Comment (tomschr) [inline doc/source/building/working_with_images/setup_yast_on_first_boot.rst]: YAST -> YaST
(I only use it once, as there are many other occurances)

File: doc/source/building/working_with_images/setup_yast_on_first_boot.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    utility at first boot of the image
+
+To be able to use YAST in a non interactive way, you need to
```
Comment (tomschr) [inline doc/source/building/working_with_images/setup_yast_on_first_boot.rst]: remove "you need to"

File: doc/source/building/working_with_images/setup_yast_on_first_boot.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+create a YAST profile which tells it what to do. To create the
```
Comment (tomschr) [inline doc/source/building/working_with_images/setup_yast_on_first_boot.rst]: Maybe replace "it" with "you"?

File: doc/source/building/working_with_images/setup_yast_on_first_boot.rst [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+   .. code:: bash
+
+       cd IMAGE_DESCRIPTION_DIRECTORY
+       mkdir -p root/etc/YaST2
+       cp PROFILE_FILE root/etc/YaST2/firstboot.xml
+
+3. Copy and activate the YAST firstboot template.
+   This is done by the following instructions which needs to be written
+   into the KIWI :file:`config.sh` which also lives in the image
```
Comment (tomschr) [inline doc/source/building/working_with_images/setup_yast_on_first_boot.rst]: "which also lives" -> "which is stored"?

**Comment by schaefi** [comment]
Hi Tom, hope you feel better ?

I just continued with this one to get new packages out. Of course I will include your changes. Thanks much

**Comment by schaefi** [comment]
I have updated the docs and pushed the changes to master

**Comment by tomschr** [comment]
> hope you feel better ?

Still recovering from the virus attack (as most people at the moment). :wink: 

> I have updated the docs and pushed the changes to master

Great, thanks! :+1:

**Comment by schaefi** [comment]
>    Still recovering from the virus attack (as most people at the moment).

Hrm, yes take your time to fully recover. I think at my place it's
too cold for viruses, -19° today in the morning *brrr*

>      I have updated the docs and pushed the changes to master
> 
>    Great, thanks! 👍

And it's live already:

   http://suse.github.io/kiwi/building/working_with_images/setup_yast_on_first_boot.html

I like our docs and I really think the pages are helpful :)
Thanks much

**Pull request merged**